src/navigations/KeyboardWrapper.tsx

```diff
@@ -12,10 +12,7 @@ const KeyboardWrapper = ({children}: Props): JSX.Element => {
   const insets = useSafeAreaInsets();
 
   return (
-    <KeyboardAvoidingView
-      style={{flex: 1}}
-      keyboardVerticalOffset={insets.top}
-      behavior={isIos ? 'padding' : undefined}>
+    <KeyboardAvoidingView style={{flex: 1}} behavior={isIos ? 'padding' : undefined}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of the `keyboardVerticalOffset` prop might cause issues with the keyboard overlaying content on iOS devices. This property was likely used to offset the keyboard's height by the safe area inset at the top of the screen, preventing it from covering important UI elements. If this change is intentional and you've tested that it doesn't cause any problems in your specific use case, then it's fine. Otherwise, consider reintroducing the `keyboardVerticalOffset` prop.

```diff
- <KeyboardAvoidingView style={{flex: 1}} behavior={isIos ? 'padding' : undefined}>
+ <KeyboardAvoidingView style={{flex: 1}} keyboardVerticalOffset={insets.top} behavior={isIos ? 'padding' : undefined}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->